tensorflow_addons/callbacks/tests/avg_model_checkpoint_test.py

```diff
@@ -0,0 +1,206 @@
+import pytest
+import tensorflow as tf
+from tensorflow import keras
+from tensorflow_addons.callbacks import AverageModelCheckpoint
+from tensorflow_addons.optimizers import MovingAverage
+import os
```

Comment:
`os` should be imported at the top. Please refer to [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) for more information